package.json

```diff
@@ -41,7 +41,7 @@
     "json": "^9.0.4",
     "linebreak": "0.3.0",
     "raw-loader": "^0.5.1",
-    "scratch-render-fonts": "git+https://github.com/LLK/scratch-render-fonts.git",
+    "scratch-svg-renderer": "git+https://github.com/LLK/scratch-svg-renderer.git",
```

Comment:
I think this is okay for now, but we should probably have scratch-svg-renderer deploy to npm like the other repos eventually.

Reply:
Tracking this in LLK/scratch-svg-renderer#1